CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# How to contribute to Medley Interlisp
```

Comment:
Since this is in the Maiko repository, I feel like this should have some indication here.